tests/network-tests/src/flows/workingGroup/leaderSetup.ts

```diff
@@ -61,6 +63,15 @@ async function leaderSetup(
   assert.notEqual(hiredLead, undefined, `${group} group Lead was not hired!`)
   assert(hiredLead!.role_account_id.eq(leadKeyPair.address))
 
+  // setup (ensure) CLI connection to node and query node
+
+  await cli.setApiUri()
+
+  await cli.setQueryNodeUri()
```

Comment:
This doesn't seem like a good place for this setup

Reply:
CLI init has been removed from there.